pkg/insights/insightsclient/insightsclient.go

```diff
@@ -66,6 +70,32 @@ func New(client *http.Client, maxBytes int64, metricsName string, authorizer Aut
 	}
 }
 
+func (c *Client) clientTransport() http.RoundTripper {
+	// default transport, proxy from env
+	clientTransport := &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+		Dial: (&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
+		}).Dial,
+		TLSHandshakeTimeout: 10 * time.Second,
+		DisableKeepAlives:   true,
+	}
+
+	// get the cluster proxy trusted CA bundle in case the proxy need it
+	rootCAs, err := c.GetTrustedCABundleFunc()
+	if err != nil {
+		klog.Errorf("Failed to get proxy trusted CA: %v", err)
+		return transport.DebugWrappers(clientTransport)
```

Comment:
This line can be left out, which I would prefer, as it reduces the number of places too look at when searching for what the function returns (+ reduces duplicate code)